resources/scss/ceres/views/Templates/BasketPreview/_basket-preview.scss

```
@@ -130,3 +130,11 @@
         }
     }
 }
+
+.basket-open {
+    .header-container {
```

```suggestion
    [data-header-offset] {
```

Auf diese Weise ist sichergestellt, dass derselbe Selektor für die Berechnung als auch für das Styling verwendet wird.

---

guter Einwand